cfgov/jinja2/v1/ask-cfpb/landing-page.html

```diff
@@ -1,43 +1,88 @@
+{% import 'atoms/partially-styled-link.html' as partially_styled_link with context %}
+{% import 'macros/category-icon.html' as category_icon %}
 {% import 'templates/render_block.html' as render_block with context %}
-{% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
+
 
 {% block content_main_modifiers -%}
     {{ super() }} content__flush-bottom
 {%- endblock %}
 
-{% block hero -%}
-    {% for block in page.header -%}
-        {% if 'hero' in block.block_type %}
-            {{ render_stream_child(block) }}
-        {% endif %}
-    {%- endfor %}
-{% endblock %}
+{% block content %}
 
-{% block content_main %}
     {% for block in page.header -%}
-        {% if block.block_type != 'hero' %}
-            {{ render_block.render(block, loop.index) }}
-        {% endif %}
-    {%- endfor %}
-    {% for block in page.cards -%}
-            {{ render_block.render(block, loop.index) }}
+        {{ render_block.render(block, loop.index) }}
     {%- endfor %}
-    {% for block in page.content -%}
-        {% if block.block_type == 'feedback' %}
-            {{- form_block.render(block, 'content', loop.index0) -}}
-        {% else %}
-            {{ render_block.render(block, loop.index) }}
-        {% endif %}
-    {%- endfor %}
-{% endblock %}
 
-{% block content_sidebar_modifiers -%}
-    o-sidebar-content
-{%- endblock %}
+    <main class="content" id="main" role="main">
+        <div class="wrapper content_wrapper ask-landing-page">
+            <div class="content-l">
+                <section class="content_main content-l_col content-l_col-1">
+                    <h2 class="u-show-on-mobile">Browse questions by category</h2>
+                    <ul class="u-show-on-mobile category_list list list__links u-mb0">
+                        {% for category in categories %}
+                            <li class="list_item">
+                                <a class="list_link jump-link jump-link__right" 
+                                   href="{{slugurl('category-' + category.slug)}}">
+                                    <span class="jump-link_text">
+                                        {{ category_icon.render(category.name) }}
+                                        {{ category.name }}
+                                    </span>
+                                </a>
+                            </li>
+                        {% endfor %}
+                    </ul>
 
-{% block content_sidebar scoped -%}
-    {{ streamfield_sidefoot.render(page.sidefoot) }}
-{%- endblock %}
+                    <div class="block block__flush-top block__flush-bottom question-categories">
+                        <div class="u-hide-on-mobile">
+
+                            <div class="m-equal-height_container content-l">
+
+                                {% for category in categories %}
+                                    <div class="content-l_col content-l_col-1-3 m-equal-height_col">
```

Comment:
If you use `content-l_col-1-3__on-lg` here instead of `content-l_col-1-3` you wouldn't have to overwrite the layout in the less file. I forget if there's a `content-l_col-1-2__on-med`, but this would be a great opportunity to add that and page specific avoid overwrites.